salt/modules/openbsdpkg.py

```diff
@@ -341,3 +341,68 @@ def purge(name=None, pkgs=None, **kwargs):
         salt '*' pkg.purge pkgs='["foo", "bar"]'
     '''
     return remove(name=name, pkgs=pkgs, purge=True)
+
+
+def upgrade_available(name):
+    '''
+    Check whether or not an upgrade is available for a given package
+
```

Comment:
Can you add some `.. versionadded:: Fluorine` tags to this function and the new function below?

Reply:
Sure thing!